### DISCUSSION TITLE
Is Next.js 10 supported?

### DISCUSSION THREAD

**Discussion by lintonye** [codesandbox/codesandbox-client#5221] [Q&A] [answered]
I got a 502 error when trying to open [this sandbox](https://codesandbox.io/s/github/lintonye/smileys-in-motion/tree/smileysinmotion.com/smileysinmotion.com). It's Next.js 10 + TailwindCSS 2.

**Comment by SaraVieira** [comment, marked as answer]
Hey

The issue, in this case, is tailwind 2 as requires node 12 and we default to node 10.

You can now change it in the sandbox.config.json

Here is an example:
https://codesandbox.io/s/friendly-greider-dpk2v?file=/sandbox.config.json

We will deploy the docs and announce this feature today or next week

**Reply by lintonye** [reply]
Thanks Sara! I changed the node version but how do I restart the container? I refreshed page and it still show 502.

**Reply by lintonye** [reply]
nvm, I found it in the "Server Controls" tab. 👍

**Reply by SaraVieira** [reply]
Glad it worked :)
